### PR TITLE
Remove nuxt playground

### PR DESCRIPTION
**Pull request by m0ksem**
Removed playground. Use sandbox:nuxt instead.

### CODE REVIEW

**Comment by rustem-nasyrov** [comment]
@m0ksem, isn't playground required for the Nuxt modules? I checked that a lot of modules have a playground.

**Comment by m0ksem** [comment]
> @m0ksem, isn't playground required for the Nuxt modules? I checked that a lot of modules have a playground.

I don't think it is required. It is a good practice to have a playground to check how module works. We have this playground in sandbox, so we don't need to have two playgrounds IMO. It doesn't impact the module in terms of types, build or whatever.

File: packages/nuxt/package.json [hunk 1]
```diff
@@ -24,11 +24,7 @@
   ],
   "scripts": {
     "build": "nuxt-module-build",
-    "prepack": "nuxt-module-build",
-    "dev": "nuxi dev playground",
-    "dev:build": "nuxi build playground",
-    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
-    "dependenciesInstalled": "node ./scripts/stud-nuxt.mjs"
+    "prepack": "nuxt-module-build"
```
Comment (rustem-nasyrov) [inline packages/nuxt/package.json:27]: Aren't they equal?

Comment (rustem-nasyrov) [reply packages/nuxt/package.json:27]: I mean `build` and `prepack` are running the same command.

Comment (m0ksem) [reply packages/nuxt/package.json:27]: prepack is used by npm and runs before npm publish. Build is used by us to get thing builded.

**Review by rustem-nasyrov — APPROVED** [review]

**Pull request merged**